### DISCUSSION TITLE
"Formal" Surface Go 4 support?

### DISCUSSION THREAD

**Discussion by esm7** [linux-surface/linux-surface#1342] [Q&A] [answered]
Hi,
I found [this](https://github.com/linux-surface/linux-surface/discussions/1269) great discussion on the relatively-new SG4, but the device is not yet listed as a linux-surface supported device.
Are still gaps to close I may have misunderstood from that discussion?
What's missing to enlist SG4 as supported, and have the (very relative) confidence that if I buy one, it will likely work roughly as well as my beloved Linux-powered SG2?

**Comment by StollD** [comment, marked as answer]
> What's missing to enlist SG4 as supported

Someone who owns it and adds the working and not working features to the wiki. 

There really is no such thing as "formal support", because we just try to fix whats broken. There are devices on the wiki that don't need the surface kernel at all, and there are plenty of devices that are "supported" but have more or less important features missing because nobody managed to figure them out yet.

The list really just serves as a reference for others to see what they can expect.

> if I buy one, it will likely work roughly as well as my beloved Linux-powered SG2?

The Go's have a pretty good track record for not following the weird™ things that Microsoft likes to do on the Pros. However on the Go 4, Microsoft seems to use UFS storage instead of NVME which is more a thing in the mobile world. So you might have trouble finding a distribution that enables the neccessary drivers on their x86 builds.
